mozetl/taar/taar_amowhitelist.py

```diff
@@ -64,12 +64,24 @@ def transform(self, json_data):
 
         new_data = {}
         for guid in json_data.keys():
```

Comment:
You can probably change this to `for guid, addon_data in json_data.items():` to be more succinct.